{"uuid": "8cdb4f77-f38e-403f-bbc8-4c6b6493ec2a", "children": ["a1773261-310a-4ade-9aa2-2d4a570d461d"], "befores": [{"name": "create_order", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: Element <button class=\"Modal_modal__close_modified__3V5XS Modal_modal__close__TnseK\" type=\"button\"> is not clickable at point (1266,218) because another element <div class=\"Modal_modal_overlay__x2ZCr\"> obscures it; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#elementclickinterceptedexception\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nElementClickInterceptedError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:351:5\nwebdriverClickElement@chrome://remote/content/marionette/interaction.sys.mjs:175:11\ninteraction.clickElement@chrome://remote/content/marionette/interaction.sys.mjs:134:11\nclickElement@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:327:29\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:210:31\n\n", "trace": "  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 929, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\tests\\conftest.py\", line 88, in create_order\n    homepage.click_cross_button_order_confirm()\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\pages\\home_page.py\", line 42, in click_cross_button_order_confirm\n    self.click(self.locators.CROSS_BUTTON_ORDER_CONFIRM)\n    ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\pages\\base_page.py\", line 13, in click\n    self.wait.until(expected_conditions.element_to_be_clickable(element_locator)).click()\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 120, in click\n    self._execute(Command.CLICK_ELEMENT)\n    ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 573, in _execute\n    return self._parent.execute(command, params)\n           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 454, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Выбираем случайную булку", "status": "passed", "start": 1754739260547, "stop": 1754739260550}, {"name": "Выбираем случайный ингредиент", "status": "passed", "start": 1754739260550, "stop": 1754739260554}, {"name": "Перетаскиваем ингредиент в корзину", "status": "passed", "parameters": [{"name": "ingredient", "value": "<selenium.webdriver.remote.webelement.WebElement (session=\"b88a2a94-ad5c-408f-b547-a5f22e84b4e5\", element=\"a755f6d6-c665-44c8-99bc-d07c4499aa0d\")>"}], "start": 1754739260554, "stop": 1754739260581}, {"name": "Перетаскиваем ингредиент в корзину", "status": "passed", "parameters": [{"name": "ingredient", "value": "<selenium.webdriver.remote.webelement.WebElement (session=\"b88a2a94-ad5c-408f-b547-a5f22e84b4e5\", element=\"dcadc440-fa29-4991-9de0-b892617d0a49\")>"}], "start": 1754739260581, "stop": 1754739260608}, {"name": "Нажимаем на кнопку Оформить заказ", "status": "passed", "start": 1754739260608, "stop": 1754739260686}, {"name": "Ждем появления высплавающего окна с подтверждением заказа", "status": "passed", "start": 1754739260687, "stop": 1754739260701}, {"name": "Ждем появления номера созданного заказа", "status": "passed", "start": 1754739260702, "stop": 1754739263332}, {"name": "Получаем номер созданного заказа", "status": "passed", "start": 1754739263332, "stop": 1754739263355}, {"name": "Нажимаем на крестик в всплывающем окне подтверждения заказа", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: Element <button class=\"Modal_modal__close_modified__3V5XS Modal_modal__close__TnseK\" type=\"button\"> is not clickable at point (1266,218) because another element <div class=\"Modal_modal_overlay__x2ZCr\"> obscures it; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#elementclickinterceptedexception\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nElementClickInterceptedError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:351:5\nwebdriverClickElement@chrome://remote/content/marionette/interaction.sys.mjs:175:11\ninteraction.clickElement@chrome://remote/content/marionette/interaction.sys.mjs:134:11\nclickElement@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:327:29\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:210:31\n\n", "trace": "  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\pages\\home_page.py\", line 42, in click_cross_button_order_confirm\n    self.click(self.locators.CROSS_BUTTON_ORDER_CONFIRM)\n    ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\pages\\base_page.py\", line 13, in click\n    self.wait.until(expected_conditions.element_to_be_clickable(element_locator)).click()\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 120, in click\n    self._execute(Command.CLICK_ELEMENT)\n    ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 573, in _execute\n    return self._parent.execute(command, params)\n           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 454, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1754739263355, "stop": 1754739263386}], "start": 1754739260547, "stop": 1754739263391}], "start": 1754739260547, "stop": 1754739263554}